packages/core/src/components/popover/popover.tsx

```diff
@@ -560,7 +557,7 @@ export class Popover extends AbstractComponent<IPopoverProps, IPopoverState> {
 
     private updateTether() {
         if (this.state.isOpen && !this.props.inline && this.popoverElement != null) {
-            const { constraints, position, tetherOptions, useSmartPositioning } = this.props;
+            const { constraints, position, tetherOptions = {}, useSmartPositioning } = this.props;
```

Comment:
so here's my thinking. 

1. default props are great for special _values_, like `false` or `Position.BOTTOM`, but they're not good for objects. for instance, #973 fixed a subtle bug due to the same `{}` instance being reused _for every instance_. i'm now leaning towards omitting `{}` and `[]` from default props as a rule.

2. there's a warning for using this prop `tetherOptions` with `inline` and it's much easier to detect prop presence if the default value is `undefined`. with the previous `{}` there was no good cheap way to determine if the user had/not specified the prop.